apple/internal/local_provisioning_profiles.bzl

```diff
@@ -0,0 +1,80 @@
+"""
+Bazel rules for using locally installed provisioning profiles for debug builds
+"""
+
+def _local_provisioning_profiles(repository_ctx):
+    repository_ctx.symlink(
```

Comment:
I think making this symlink might obviate the need to make external. Caveat: if it doesn't exist but somehow exists in the future.

You may want to run `mkdir -p` on this dir to handle that and in general if it may not exist on a new system